1.9/metrics/datadog.md

```diff
@@ -0,0 +1,135 @@
+---
+post_title: Datadog Metrics for DC/OS
+feature_maturity: preview
+menu_order: 3.5
+---
+
+The Datadog metrics plugin for DC/OS supports sending metrics from the DC/OS metrics service on master and agent hosts to a Datadog agent for shipping to DatadogHQ.
+
+# Installation
+
+## Build the plugin
+
+**Prerequisite:** 
+- DC/OS is installed
+- [Go programming environment](https://golang.org/doc/install) <!-- dcos-metrics must be run from within the go directory -->
+- GitHub
```

Comment:
what aspect of GitHub is a prerequisite? Does the user need a GitHub account? I guess they also need git installed on their machine? Would a link be helpful?